--- cryptominisat-5.6.4/python/setup.py.in.orig	2018-08-05 08:17:15.000000000 -0500
+++ cryptominisat-5.6.4/python/setup.py.in	2018-08-24 20:21:41.000000000 -0500
@@ -117,6 +117,12 @@
 if platform.system() == 'Darwin':
     extra_link_args.append('-Wl,-rpath,'+"${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}")
 
+if platform.system() == 'Windows':
+    libname = "cryptominisat5win"
+else:
+    libname = "cryptominisat5"
+
+
 modules = dict(
     name = "pycryptosat",
     sources = ["${CMAKE_CURRENT_SOURCE_DIR}/src/pycryptosat.cpp"],
@@ -126,7 +132,7 @@
     language = "c++",
     library_dirs=['.', '${PROJECT_BINARY_DIR}/lib'],
     runtime_library_dirs=['${CMAKE_INSTALL_PREFIX}/${CMAKE_INSTALL_LIBDIR}'],
-    libraries = ['cryptominisat5']
+    libraries = [libname]
 )
 
 setup(
